data/json/items/tools.json

```diff
@@ -1,4 +1,4 @@
-[
+﻿[
```

Comment:
At a guess:
![that one](http://i.imgur.com/ULc0Kfi.png)

I don't actually use N++ for Cata myself, though, so this was just me opening the file for a test.